subprojects/testing-jvm/src/integTest/resources/org/gradle/testing/junit/JUnitCategoriesIntegrationSpec/reExecutesWhenPropertyIsChanged/src/integTest/java/org/gradle/SomeTestClass.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018 the original author or authors.
```

Comment:
These are new files and looks like they haven't been migrated from anywhere else. Should we have a modern copyright date?

Reply:
I'm amazed you noticed the dates.  Will update.